src/ui/handler/drag_rotate.js

```diff
@@ -94,7 +94,7 @@ class DragRotateHandler {
         if (this._ignoreEvent(e)) return;
         if (this.isActive()) return;
 
-        window.document.addEventListener('mousemove', this._onMove);
+        window.document.addEventListener('mousemove', this._onMove, {capture: true});
```

Comment:
This was necessary for the "DragRotateHandler prevents mousemove events from firing during a drag" test to pass.